barretenberg/cpp/src/barretenberg/client_ivc/client_ivc.cpp

```diff
@@ -92,8 +92,8 @@ void ClientIVC::perform_recursive_verification_and_databus_consistency_checks(
 
     // Recursively verify the merge proof for the circuit being recursively verified
     // TODO(https://github.com/AztecProtocol/barretenberg/issues/950): handle pairing point accumulation
-    [[maybe_unused]] AggregationObject pairing_points =
-        GoblinVerifier::recursive_verify_merge(circuit, verifier_inputs.merge_proof);
+    RecursiveMergeVerifier merge_verifier{ &circuit };
```

Comment:
No logic change here, just explicitly using the RecursiveMergeVerifier instead going through a method on Goblin that no longer exists.